pkg/log/status/status.go

```diff
@@ -14,18 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package log contains logging related functionality
-package log
+// Package status
+package status
```

Comment:
this also didn't really belong in the main log package and was the bulk of this diff.

next up we will have to re-arrange how this interacts with the logger, have a few ideas for that, but it will be easier to work out once we start actually switching the logger...